src/module-elasticsuite-catalog-optimizer/Controller/Adminhtml/Optimizer/SaveDuplicate.php

```diff
@@ -0,0 +1,92 @@
+<?php
+/**
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade Smile ElasticSuite to newer
+ * versions in the future.
+ *
+ * @category  Smile
+ * @package   Smile\ElasticsuiteCatalogOptimizer
+ * @author    Dmytro ANDROSHCHUK <dmand@smile.fr>
+ * @copyright 2020 Smile
+ * @license   Open Software License ("OSL") v. 3.0
+ */
+
+namespace Smile\ElasticsuiteCatalogOptimizer\Controller\Adminhtml\Optimizer;
+
+use Smile\ElasticsuiteCatalogOptimizer\Api\Data\OptimizerInterface;
+use Smile\ElasticsuiteCatalogOptimizer\Controller\Adminhtml\AbstractOptimizer as OptimizerController;
+
+/**
+ * Optimizer Adminhtml Save duplicate controller.
+ *
+ * @category Smile
+ * @package  Smile\ElasticsuiteCatalogOptimizer
+ * @author   Dmytro ANDROSHCHUK <dmand@smile.fr>
+ */
+class SaveDuplicate extends OptimizerController
```

Comment:
There should not be any new controller to save a duplicated optimizer. Only the legace `Save` should be responsible for saving.